src/project/internal/notationproject.cpp

```diff
@@ -593,9 +593,9 @@ mu::Ret NotationProject::makeCurrentFileAsBackup()
     }
 
     io::path_t backupFilePath = filePath + "~";
-    ret = fileSystem()->move(filePath, backupFilePath, true);
+    ret = fileSystem()->copy(filePath, backupFilePath, true);
     if (!ret) {
-        LOGE() << "failed to move from: " << filePath << ", to: " << backupFilePath;
+        LOGE() << "failed to copy: " << filePath << " to: " << backupFilePath;
         return ret;
```

Comment:
while I agree this is better, are you sure it doesn't break regular save, which appears to try to move from "xxxx_saving" to "xxxx", which is likely to fail if xxxx stil exists?

Reply:
Oh I didn't think about that. Let me check

Reply:
>are you sure it doesn't break regular save, which appears to try to move from "xxxx_saving" to "xxxx", which is likely to fail if xxxx still exists?

It shouldn't because we call copy with replace = true. Anyway, I asked @DmitryArefiev  to check both Save and Save as

Reply:
When I checked it nothing broke. But I thought my solution might not be the correct fix :)